### ISSUE TITLE
Document the fact the two-factor isn't supported.

### ISSUE THREAD

**Issue by pedramamini** [IQAndreas/github-issues-import#46]
Took me a few to figure out that's why authentication was failing.

**Comment by bikeonastick** [comment]
you can still use this by creating a Personal Access Token in github and substitute that for your password. I have two-factor auth set up but was able to use this script with my Personal Access Token.

**Comment by pedramamini** [comment]
Good call... but perhaps then document the fact that a PAT should be used :-) will leave it up to you to close the issue out.

**Comment by bikeonastick** [comment]
heh... i am not part of the project, but i may submit a pull request to add it to the documentation (if you don't beat me to it).

--rt

**Comment by pedramamini** [comment]
Far too lazy for that... but enjoy :-) While you're at it:

```
https://github.com/IQAndreas/github-issues-import/issues/47
```

It's not _as_ easy as just pasting those lines at the top. I suspect the authors would like to add a new command line switch.
